src/bufr2ncObsTypes.py

```diff
@@ -982,7 +993,9 @@ def extract_bufr(self, bufr):
                                      mask=BangleBvals.mask[m+2,irep])  # bending angle
                         BNDA_err = np.ma.array(BangleBvals[m+4,irep],
                                      mask=BangleBvals.mask[m+4,irep])  # bending angle error
-                        break
+                        #break
+# CSD - fortran does not exit after first zero-frequency is found. I think there is only 
+# one, but just in case, don't break? 
```

Comment:
The break was put in to skip over processing the non-zero frequency cases when the zero frequency case came before the others. However, this is assuming that only one zero frequency case exists. In the gps bufr file that I used this assumption is good, and it looked like for all of the ROSEQ2 sections there was either one replication (zero frequency) or three replications (two non-zero and one zero). If the max number of replications we expect to see is actually three, then leaving out the break shouldn't hurt performance and I'm good with leaving the break statement out.